Source/Core/Heap.js

```diff
@@ -111,7 +111,7 @@ define([
         var data = this._data;
         var candidate = -1;
 
-        while (true) {
+        while (true) { // eslint-disable-line no-constant-condition
```

Comment:
Already noted in #5308 for Cesium3DTileBatchTable.js and could probably apply here as well.

CC @lilleyse